### PR TITLE
Translated the license page into German

### PR DESCRIPTION
**Pull request by john2ksonn**
I translated the upper paragraph of the license page into German and updated the yaml header accordingly.

### CODE REVIEW

**Comment by john2ksonn** [comment]
I created a separate PR for this so it will be easier to manage

File: _i18n/de/license.md [hunk 1]
```diff
@@ -0,0 +1,1187 @@
+---
+title: "Lizensen"
```
Comment (kuchosauronad0) [inline _i18n/de/license.md]:
```suggestion
title: "Lizenzen"
```

File: _i18n/de/license.md [hunk 2]
```diff
@@ -0,0 +1,1187 @@
...
+layout: qmk
+permalink: /de/license/
+---
+
+# Lizensen
```
Comment (kuchosauronad0) [inline _i18n/de/license.md]:
```suggestion
# Lizenzen
```

File: _i18n/de/license.md [hunk 3]
```diff
@@ -0,0 +1,1187 @@
...
+
+Ein Großteil von QMK steht unter der GPL v2, aber QMK beinhaltet ebenso Software-Elemente die unter anderen Lizensen stehen, die alle hier kurz beschrieben werden. Der ganze Text der jeweiligeb Lizensen können Sie in dem Code-Repository und unten finden
```
Comment (kuchosauronad0) [inline _i18n/de/license.md]:
```suggestion
Der Großteil von QMK steht unter der GPL v2, aber QMK beinhaltet ebenso Software-Elemente die unter anderen Lizenzen stehen, die alle hier kurz beschrieben werden. Der ganze Text der jeweiligen Lizenzen können Sie in dem Code-Repository und auf dieser Seite im Folgenden finden.
```

**Comment by john2ksonn** [comment]
Thank you, @kuchosauronad0 :+1:

**Review by kuchosauronad0 — APPROVED** [review]
Looks good to me :) Thanks

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**